lib/types.ts

```diff
@@ -21,3 +21,23 @@ export enum OsReleaseFilePath {
   RedHat = "/etc/redhat-release",
   Oracle = "/etc/oracle-release",
 }
+
+export interface ManifestFile {
+  name: string;
+  path: string;
+  contents: string;
+}
+
+export interface PluginMetadata {
+  name: string;
+  runtime: string | undefined;
+  packageManager: any; // TODO
```

Comment:
Does the TODO add anything here? I would remove it and make the type more concrete when it becomes necessary. If you think the TODO should be there then we should at least describe what exactly needs doing :smiley:

Reply:
it makes it a bit more obvious that we're unhappy with the ```any``` type here, and we intend fixing it soon.
I find it self explanatory that it refers to the ```any```, because it really sticks out to me.
what do you think?

Reply:
I don't mind really but have a dislike for TODO's in the code which don't exactly say what needs doing. For you in the moment it is clear that they refer to the any type but it could also mean that some types are obsolete or need further differentiation or maybe the name is just not ideal in the moment. The meaning becomes ambiguous over time.

Instead of TODOs in the code I would track this as a technical debt task in Jira. This is a good example of technical debt which needs addressing at some point. In magma we have the "Tech-debt & maintenance" epic for this.

Reply:
a great explanation, I'm removing the TODOs. thanks!